### ISSUE TITLE
Normalization problem of model using EfficientNet-Lite backbone.

### ISSUE THREAD

**Issue by RangiLyu** [nihui/ncnn-android-nanodet#1]
According to Google‘s EfficientNet Lite imagenet training setting, the means and stds of normalization are [127.0, 127.0, 127.0] and [128.0, 128.0, 128.0]. I trained NanoDet-EfficientNet with the same values.
https://github.com/RangiLyu/nanodet/blob/b8763e75e192fe4521db6e93344fabc0544ec0e9/config/EfficientNet-Lite/nanodet-EfficientNet-Lite0_320.yml#L75

The values of these lines are quite different from the pre-trained model.
https://github.com/nihui/ncnn-android-nanodet/blob/80f9b00353c9ad00b0a49c2a76d78bfe9ba5ab55/app/src/main/jni/nanodet.cpp#L306
https://github.com/nihui/ncnn-android-nanodet/blob/80f9b00353c9ad00b0a49c2a76d78bfe9ba5ab55/app/src/main/jni/nanodet.cpp#L307

**Comment by nihui** [comment]
fixed in https://github.com/nihui/ncnn-android-nanodet/commit/4b93c1736a84c2c0109bc7ff1a92782e26892c59

Thanks for reporting.
